.gitignore

```diff
@@ -0,0 +1,94 @@
+.idea/
```

Comment:
Remove the endor filepath from all of these since they won't be in a folder called endor anymore

Reply:
Done

Reply:
👍